### ISSUE TITLE
[Nerdlet Details] Display the details for integration packs

### ISSUE THREAD

**Issue by jpvajda** [newrelic/newrelic-observability-packs#135]
TBD

**Comment by aswanson-nr** [comment]
@jpvajda We display a sort of "category" for each component at the bottom of the tile. 
<img width="363" alt="Screen Shot 2021-08-23 at 1 31 58 PM" src="https://user-images.githubusercontent.com/70179303/130515461-e0bca75d-7469-4955-9ce8-0cbfc1d3338a.png">
Do we want to add a "documentation" category to it?

**Comment by jpvajda** [comment]
I don't think we should do anything with pack categories or keywords until we work that scope out.. it's something we are doing this week.
